src/components/Wallet/Asset.tsx

```diff
@@ -57,19 +72,40 @@ export default function Asset({ balance }: Props) {
     }
     const parserAmount = parseAmountToDecimal(amount);
     try {
-      const notificationId = toast(
-        <LoaderAndText text={MessageLoad.TRANSACTION} />,
-        optionsAsync,
-      );
+      const notificationId = succesLoaderMsg(MessageLoad.TRANSACTION);
       handleChangeBoolean();
       setTransaction(INITIAL_STATE);
-      await sendTransaction(secretKey, publicKey, parserAmount);
+      let signature = false;
+      if (payment === 'Albedo') {
```

Comment:
What happens if tomorrow we've to add more wallets? This condition will be bigger and not legible.